src/Plugin/EdgeKeyTypeBase.php

```diff
@@ -62,6 +62,13 @@ public function getEndpoint(KeyInterface $key): string {
     return $key->getKeyValues()['endpoint'] ?? Client::DEFAULT_ENDPOINT;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getEndpointType(KeyInterface $key): string {
+    return $key->getKeyValues()['endpoint_type'] ?? 'default';
```

Comment:
I would probably consider introducing constants for `default` and `custom` in EdgeKeyTypeInterface because I keep seeing these magic words appearing in different places of the code. Ex.: in ApigeeAuthKeyForm.

Reply:
Added the constants and updated appearances of `default` and `custom` to use them.